docs/pages/reference/cli.mdx

```diff
@@ -423,6 +423,67 @@ $ tsh proxy --port 10700 app grafana &
 $ curl http://127.0.0.1:10700/api/users
 ```
 
+### tsh proxy azure
+
+Starts a local proxy server that provides secure access to an Azure managed
+service identity endpoint. This is useful for managing access to Azure from
+custom client applications. The local proxy forwards traffic to the Teleport
+Application Service, which uses an Azure managed identity to fetch an
+authentication token from Azure.
+
+```code
+$ tsh proxy azure [<flags>]
+```
+
+The command will print the address of the local proxy server along with `export`
+commands for environment variables required to connect:
+
+```text
+Started Azure proxy on http://127.0.0.1:54330.
+To avoid port randomization, you can choose the listening port using the --port flag.
+
+Use the following credentials and HTTPS proxy setting to connect to the proxy:
+
+  export AZURE_CONFIG_DIR=/Users/myuser/.azure-teleport
```

Comment:
`AZURE_CONFIG_DIR` is now `~/.tsh/azure/<cluster>/<app-name>`, so something like:

```suggestion
  export AZURE_CONFIG_DIR=/Users/myuser/.tsh/azure/my.teleport.cluster/azure
```